### ISSUE TITLE
Add FAQ.md for frequent issues and fixes

### ISSUE THREAD

**Issue by athityakumar** [athityakumar/colorls#84]

**Comment by kaustubhhiware** [comment]
Also, a link to FAQ.md must be subsequently be added to the README, so that anyone does not have to search for such a file by themself or find a relevant issue.

**Comment by athityakumar** [comment]
Yes, this is to be linked from the README (preferably, at the beginning of Installation section).
